api/src/main/java/org/apache/brooklyn/api/objs/Configurable.java

```diff
@@ -41,6 +41,11 @@
     @Deprecated
     public <T> T setConfig(ConfigKey<T> key, T val);
 
+    /**
+     * Convenience method for {@code config().get(key)}
+     */
+    <T> T getConfig(ConfigKey<T> key);
```

Comment:
I see you just moved it up the hierarchy now, so not the time to do any changes, but the above still holds.

Can remove it from `Entity`, `Policy`, etc. in this case.